content/account-data-matching.md

```diff
@@ -145,7 +145,7 @@ pub struct AdminConfig {
 }
 ```
 
-Bilang kahalili, maaari mong gamitin ang `constraint` upang manu-manong magdagdag ng isang expression na dapat suriin sa true upang magpatuloy ang pagpapatupad. Ito ay kapaki-pakinabang kapag sa ilang kadahilanan ay hindi maaaring maging pare-pareho ang pagbibigay ng pangalan o kapag kailangan mo ng mas kumplikadong expression upang ganap na mapatunayan ang papasok na data.
+Bilang kahalili, maaari mong gamitin ang `constraint` upang manu-manong magdagdag ng isang expression na dapat evaluate to true upang magpatuloy ang pagpapatupad. Ito ay kapaki-pakinabang kung ang pagbibigay ng pangalan ay hindi konsistent o kapag kailangan mo ng mas kumplikadong expression upang ganap na mapatunayan ang papasok na data.
```

Comment:
`evaluate to` is still English

Reply:
Hello sir, Instead of evaluate to, I'll use "mag-evaluate to" para mag makes sense sya sa tag-lish. Hindi po kasi tugma kapag "suriin" yung gamit na word.